projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.html

```diff
@@ -1,21 +1,22 @@
 <tbody class="legend">
     <ng-template [ngIf]="layer.symbology.colorizer.isContinuous()">
-        <tr *ngFor="let breakpoint of layer.symbology.colorizer.getBreakpoints(); let isFirst = first">
+        <!-- measurement -->
+        <ng-container *ngIf="measurement$ | async | continuousMeasurement as measurement">
+            ( {{ measurement.measurement }} <span *ngIf="measurement.unit">in {{ measurement.unit }}</span> )
+        </ng-container>
+        <ng-container *ngIf="measurement$ | async | classificationMeasurement as measurement">
+            ( {{ measurement.measurement }} )
```

Comment:
1. Now you've put everything in braces which looks bad. Please only put the unit part in braces.

![grafik](https://user-images.githubusercontent.com/4947061/193067573-ad4643ba-b9d5-497f-b0ef-c81cda0a0a42.png)

2. There should not be whitespace before and after the brace.

3. Moreover, please add a small margin to the bottom so that it looks nice.

Reply:
I moved the braces to the unit, deleted whitespace and put 0.5rem as a bottom margin. 330864ceb13c28994b236530b8cf6744a8428b7f